.gitignore

```diff
@@ -21,3 +21,5 @@ debian/debhelper-build-stamp
 debian/delphix-platform-*
 debian/files
 debian/tmp
+delphix-platform.iml
```

Comment:
Can you add a brief comment above these to describe where these files would come from? I presume it's from some IDE?

Reply:
Done